com.microsoft.mrtk.input/Visualizers/ControllerVisualizer/ControllerVisualizer.cs

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.OpenXR;
+using Microsoft.MixedReality.Toolkit.Input.Simulation;
+using UnityEngine;
+using UnityEngine.InputSystem.Utilities;
+using UnityEngine.XR;
+using UnityEngine.XR.Interaction.Toolkit;
+using UnityInputSystem = UnityEngine.InputSystem;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Basic controller visualizer which draws a generic motion controller when one is detected
+    /// </summary>
+    [AddComponentMenu("MRTK/Input/Controller Visualizer")]
+    public class ControllerVisualizer : MonoBehaviour
+    {
+        [SerializeField]
+        [Tooltip("The XRNode on which this hand is located.")]
+        private XRNode handNode = XRNode.LeftHand;
+
+        /// <summary> The XRNode on which this hand is located. </summary>
+        public XRNode HandNode { get => handNode; set => handNode = value; }
+
+        [SerializeField]
+        [Tooltip("A fallback controller model to render in case the platform model fails to load")]
+        private GameObject fallbackControllerModel;
```

Comment:
Perhaps allow the user to leave this blank/null in case they don't want to show a fallback generic? (It'll just pass it straight to Instantiate below, which seems like it'll cause issues?)

Reply:
Added a null check.